src/github.com/couchbase/sync_gateway/db/crud.go

```diff
@@ -614,7 +620,7 @@ func (db *Database) updateDoc(docid string, allowImport bool, callback func(*doc
 
 			// Update the document struct's channel assignment and user access.
 			// (This uses the new sequence # so has to be done after updating doc.Sequence)
-			changedChannels = doc.updateChannels(channels) //FIX: Incorrect if new rev is not current!
+			changedChannels = doc.updateChannels(channelSet) //FIX: Incorrect if new rev is not current!
```

Comment:
This was more for readability and search-ability in IDE, there is an import for "github.com/couchbase/sync_gateway/channels" and there are a number of references to constants e.g.

channels.Deleted
channels.Conflict

the change to channelSet makes it clearer that these are two separate entities